### PR TITLE
Aggregator Facebook oAuth

### PR DESCRIPTION
**Pull request by bordoni**

### CODE REVIEW

File: src/resources/js/aggregator-facebook-login.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+var tribe_aggregator = tribe_aggregator || {};
+( function( $, _, ea ) {
+	'use strict';
+
+	ea.receive_message = function( event ) {
```
Comment (borkweb) [inline src/resources/js/aggregator-facebook-login.js]: I feel like this should be nestled under another level. Maybe `ea.facebook.receive_message`?

File: vendor/select2 [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 1c5c8dd94ad82e5cbecf59806890e5e0c9a17739
+Subproject commit ecc2b14d860de19b9447be3628a768217de6602c
```
Comment (borkweb) [inline vendor/select2]: Roll this back to 1c5c8dd

**Comment by bordoni** [comment]
👍

**Comment by borkweb** [comment]
:+1:

**Pull request merged**